packages/010editor.vm/tools/chocolateyinstall.ps1

```diff
@@ -3,7 +3,7 @@ Import-Module vm.common -Force -DisableNameChecking
 
 try {
   $toolName = '010Editor'
-  $category = 'Hex Editors'
+  $category = VM-Get-Category($MyInvocation.MyCommand.Definition)
```

Comment:
I think you can call `$MyInvocation.MyCommand.Definition` inside the function so that the function does not receive any  arguments:
```suggestion
  $category = VM-Get-Category()
```

Reply:
I was wrong, it seems it fails when moving it.